app/src/main/java/org/oppia/app/drawer/NavigationDrawerFragment.kt

```diff
@@ -0,0 +1,31 @@
+package org.oppia.app.drawer
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.widget.Toolbar
+import androidx.drawerlayout.widget.DrawerLayout
+import org.oppia.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+/** [NavigationDrawerFragment] to show navigation drawer. */
+class NavigationDrawerFragment : InjectableFragment() {
+
+  @Inject
+  lateinit var navigationDrawerFragmentPresenter: NavigationDrawerFragmentPresenter
```

Comment:
Ditto: This should be in single line:

`@Inject lateinit var navigationDrawerFragmentPresenter: NavigationDrawerFragmentPresenter`

Reply:
@nikitamarysolomanpvt assign it back to me once you have resolved all the comments. I still need to check the comments from @BenHenning 's side.

Reply:
Ditto: